notification/test/unit/config.spec.js

```diff
@@ -33,4 +33,56 @@ describe('::config::', () => {
     // eslint-disable-next-line global-require,import/no-dynamic-require
     return require(module)
   }
+
+  it(
+    'when removeSensitiveData is set as boolean false in config.js, ' +
+      'it should load as false value in module config',
+    () => {
+      process.env.ADYEN_INTEGRATION_CONFIG = JSON.stringify({
+        commercetools: {
+          ctpProjectKey1: {
+            clientId: 'clientId',
+            clientSecret: 'clientSecret',
+            apiUrl: 'host',
+            authUrl: 'authUrl',
+          },
+        },
+        adyen: {
+          adyenMerchantAccount1: {
+            enableHmacSignature: 'false',
+          },
+        },
+        logLevel: 'DEBUG',
+        removeSensitiveData: false,
+      })
+      const config = requireUncached('../../src/config/config')
+      expect(config.getModuleConfig()?.removeSensitiveData).to.eql(false)
+    }
+  )
+
+  it(
+    'when removeSensitiveData is set as boolean true in config.js, ' +
```

Comment:
What if it is set as String? You also going to test it?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/817/commits/b3ee93bd5b59eb314637b2e6655e1d1cce908da3